paddle/phi/ops/yaml/inconsistent/onednn_ops_extra.yaml

```diff
@@ -32,7 +32,6 @@
 
 - op : cast
   dynamic_fallback : True
-  extra_args : str mkldnn_data_type="float32"
```

Comment:
Maybe also need to modify op_compat.yaml?

Reply:
I think it's fine, cause `op_compat.yaml` usually has the maximum set, which only reports error when adding not included attr(s).